style.css

```diff
@@ -7,458 +7,1006 @@
-  list-style: none;
-  padding-bottom: 10px;
-}
-
-.abt-social li {
-  display: inline-block;
-  margin-right: 2%;
-}
-
-.abt-social li img {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  color: #7f8cff;
-  text-decoration: none;
-  margin-left: 285%;
-}
-
-.skills {
-  display: flex;
-  flex-direction: column;
-  align-items: start;
-}
-
-.arrow {
-  display: flex;
-  width: 90%;
-  justify-content: space-between;
-  align-items: center;
-  padding: 1rem 0;
-}
-
-.tech {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  margin: 15px;
-  color: #253858;
-}
-
-.languages li {
-  background: #f7f7f9;
-}
-
-.contact-container {
-  margin: 0 auto;
-  position: relative;
-  max-height: 800px;
-  margin-bottom: 50px;
-  background-color: white;
-}
-
-.form-container {
-  position: relative;
-  background-color: #6070ff;
-  margin-top: 177px;
-  border-top-left-radius: 80px;
-  display: flex;
-  flex-direction: column;
-}
-
-.back-spot {
-  position: absolute;
-  z-index: 1;
-  top: 21px;
-  right: 0;
-}
-
-.form-bottom {
-  z-index: 99;
-}
-
-.form-bottom h2 {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-top: 103px;
-  color: white;
-}
-
-.form-bottom p {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 28px;
-  margin-top: 12px;
-  margin-left: 24px;
-  margin-right: 24px;
-  color: white;
-  text-align: center;
-}
-
-.first {
-  margin: 52px 24px 0 24px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  width: 327px;
-  padding: 10px;
-}
-
-.first::placeholder {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.second {
-  margin: 20px 24px 0 24px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  width: 327px;
-  padding: 10px;
-}
-
-.second::placeholder {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.message {
-  background: #fff;
-  margin-top: 18px;
-  margin-left: 24px;
-  width: 327px;
-  padding: 15px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-}
-
-.btn {
-  margin-top: 22px;
-  margin-left: 24px;
-  padding: 12px 16px;
-  background: #fff;
-  border-radius: 8px;
-  border: none;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  cursor: pointer;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #6070ff;
-}
-
-.btn:hover {
-  background-color: #bdbdf1;
-  color: white;
-}
-
-.form-bottom hr {
-  margin: 0 auto;
-  margin-bottom: 8px;
-  width: 125px;
-  height: 5px;
-  background-color: black;
-  margin-top: 36px;
-  border-radius: 10px;
-  border: none;
+/* media query for mobile version  */
```

Comment:
Hi @huseyinbiyik thanks for the hardwork reviewing my codes.

Yes, while making the mediaquery for desktop version I have somehow messed the classes of mobile version.
I still just to submitted the desktop version while I can find out what made classes in mob first got messed up.

I have one curious question though. Is mobile version included in the core requirements for this project??

As I have double checked, this project will only be based on desktop version?!! Otherwise I would refrain from submitting.
If you say I will not submit again before fixing all the versions.

Additional advice on mobile version will also help a lot 🙂 thanks.

Waiting for your advice.

Thanks.

Reply:
Hello @Zafron047,

We are building our projects on top of each other, so we are developing them instead of breaking the previous ones and I think it is common sense. But if you still think the review is unfair please ask for second opinion with [this form](https://airtable.com/shrQAqnBwek5a0O0s). 👍

Reply:
@huseyinbiyik I don't think the code was unfair. 
You have done a thorough inspection which is very useful for me. And I appreciate it.

However, I just followed the necessary requirements documented for submission.

I need to fully understand the requirements again since we have limited reviews keeping it in mind if it will get rejected outside the desktop version again.

And 🤔, as I came to know both versions will be checked in next block's project where we will have to debug this project?

Further advice please.
Should I reuse the previous css stylings for the mobile version, And only change the desktop stylings to fix both issues?

Reply:
Fix Mobile version done.